packages/addons/service/hyperion.ng/source/bin/hyperiond.start

```diff
@@ -21,14 +21,14 @@
 
 oe_setup_addon service.hyperion.ng
 
-if [ ! -f "$ADDON_HOME/hyperion.config.json" ]; then
+if [ ! -f "$ADDON_HOME/config/hyperion.config.json" ]; then
   mkdir -p $ADDON_HOME
-  cp $ADDON_DIR/config/hyperion.config.json.sample $ADDON_HOME/hyperion.config.json
+  cp $ADDON_DIR/config/hyperion.config.json.sample $ADDON_HOME/config/hyperion.config.json
 fi
 
-exec hyperiond $ADDON_HOME/hyperion.config.json &
+exec hyperiond --rootPath $ADDON_HOME hyperion.config.json &
```

Comment:
Missing slash? Or is that intended?

Reply:
It is intended, there are two arguments now. 
root path, plus the configuration file name.
I tested the following use cases:
 - new user to hyperion.ng (no previous configuration/userdata)
 - user that updates from previous versions of hyperion.ng (hyperion.config.json does exist)

All are working